themes/absimple/README.md

```diff
@@ -0,0 +1,17 @@
+# ab+simple theme
+
+this theme needs a Powerline-patched font to render correctly. i recommend: https://github.com/adi1090x/termux-style
+
+a colorful theme based on agnoster theme, made shorter and simpler.
+
+added title support for termux and random dates for fun.
+
+the aim of this theme is to only show you *relevant* information.
+
+ie: it will only show git information when in a git working directory.
+
+also, everything from the current user and hostname to whether the last call exited with an error or whether background jobs are running in this shell will all be displayed automatically when appropriate.
+
+![bash screenshot](absimple-bash.png)
+![termux screenshot](absimple-termux.png)
+![random screenshot](absimple-random.png)
```

Comment:
We want to keep the repository size in a reasonable size. We wouldn't like to add multiple images for a theme nor an image that has over 100 KiB.